### ISSUE TITLE
Redemption of escrowed payments not working in many cases

### ISSUE THREAD

**Issue by nityas** [celo-org/celo-monorepo#5274]
### Expected Behavior

un-redeemed escrowed payments should always be able to be reclaimed

### Current Behavior

see logs from rene sent 10/5: payments unable to be reclaimed

**Comment by tarikbellamine** [comment]
I think we need more details on this before it's actionable. @dominiquejane do you have any logs or additional information you could share?

**Comment by dominiquejane** [comment]
@tarikbellamine  Unfortunately, I don't have logs, it seems like Rene's logs are not actually being attached when he tried to send them. He tried resending but they still wouldn't attach. We may have to tackle this log issue before we can look into the reclaiming issue. Note, Will also faced issues with attaching logs.

**Comment by nityas** [comment]
Closing, duplicate of #5822
